server.js

```diff
@@ -1,20 +1,106 @@
 const express = require("express");
 const cors = require("cors");
+const uuid = require("uuid");
+const PORT = 9000;
 
 const app = express();
 
 app.use(express.json());
 app.use(cors());
 
+// TODO add your routes and helper functions here
+
+// const listener = app.listen(process.env.PORT, function () {
+//   console.log("Your app is listening on port " + listener.address().port);
+// });
+app.listen(PORT, function () {
+  console.log(`Server is listening on ${PORT}. Ready to accept requests!`);
+});
+
 //Use this array as your (in-memory) data store.
 const bookings = require("./bookings.json");
+const { response, request } = require("express");
 
-app.get("/", function (request, response) {
-  response.send("Hotel booking server.  Ask for /bookings, etc.");
+// to read all the bookings
+app.get("/bookings", function (request, response) {
+  response.json(bookings);
 });
 
-// TODO add your routes and helper functions here
+// Read one booking, specified by an ID
+app.get("/bookings/:id", (request, response) => {
+  const foundId = bookings.some(
+    (eachbooking) => eachbooking.id === parseInt(request.params.id)
+  );
+  console.log(foundId);
+  if (foundId) {
+    response
+      .status(200)
+      .json(
+        bookings.filter(
+          (eachmessage) => eachmessage.id === parseInt(request.params.id)
+        )
+      );
+  } else {
+    response
+      .status(404)
+      .json({
+        message: `No bookings with the id of ${request.params.id} is found`,
+      });
+  }
+});
+
+//  to create bookings & level :02 
+
+
+app.post("/createNewBookings", (request, response) => {
```

Comment:
remember to use the same route you already have - for example should be able to POST and GET from `/bookings` rather than creating a new route